.circleci/config.yml

```diff
@@ -591,13 +628,253 @@ workflows:
+          testmodule: aws-android-sdk-rekognition-test
+          requires:
+            - pre_integrationtest    
+          filters:
+            branches:
+              only:
+                - master
+                - develop
+                - runintegrationtest  
+      - integrationtest:
+          name: s3
+          testmodule: aws-android-sdk-s3-test
+          requires:
+            - pre_integrationtest    
+          filters:
+            branches:
+              only:
+                - master
+                - develop
+                - runintegrationtest  
+      - integrationtest:
+          name: sdb
+          testmodule: aws-android-sdk-sdb-test
+          requires:
+            - pre_integrationtest    
+          filters:
+            branches:
+              only:
+                - master
+                - develop
+                - runintegrationtest                                   
+      - integrationtest:
+          name: ses
+          testmodule: aws-android-sdk-ses-test
+          requires:
+            - pre_integrationtest    
+          filters:
+            branches:
+              only:
+                - master
+                - develop
+                - runintegrationtest                                   
+      - integrationtest:
+          name: sns
+          testmodule: aws-android-sdk-sns-test
+          requires:
+            - pre_integrationtest    
+          filters:
+            branches:
+              only:
+                - master
+                - develop
+                - runintegrationtest   
+      - integrationtest:
+          name: sqs
+          testmodule: aws-android-sdk-sqs-test
+          requires:
+            - pre_integrationtest    
+          filters:
+            branches:
+              only:
+                - master
+                - develop
+                - runintegrationtest   
+
+      - integrationtest:
+          name: transcribe
+          testmodule: aws-android-sdk-transcribe-test
+          requires:
+            - pre_integrationtest    
+          filters:
+            branches:
+              only:
+                - master
+                - develop
+                - runintegrationtest   
+      - integrationtest:
+          name: translate
+          testmodule: aws-android-sdk-translate-test
+          requires:
+            - pre_integrationtest    
+          filters:
+            branches:
+              only:
+                - master
+                - develop
+                - runintegrationtest   
+      - integrationtest:
+          name: mobile-client
+          testmodule: aws-android-sdk-mobile-client
+          requires:
+            - pre_integrationtest    
+          filters:
+            branches:
+              only:
+                - master
+                - develop
+                - runintegrationtest  
+      - post_integrationtest:
+          requires:
+            - mobile-client
+            - translate
```

Comment:
We need to add kinesisvideo and kinesisvideoarchivedmedia to the list

Reply:
it seems we don't have test module for kinesisvideo and kinesisvideoarchivedmedia  so far